### ISSUE TITLE
Uncaught TypeError: Cannot read property 'state' of undefined

### ISSUE THREAD

**Issue by grimen** [zalmoxisus/redux-devtools-extension#47]
Stacktrace:

```
Uncaught TypeError: Cannot read property 'state' of undefined
unliftState @ instrument.js?1fd6:307
getState @ instrument.js?1fd6:327
getLocationState @ index.js?efd0:103
middleware.listenForReplays @ index.js?efd0:105
configureStore @ createReduxStore.js?0024:51
(anonymous function) @ index.jsx?2e9e:10
(anonymous function) @ bundle.js:2758__webpack_require__ @ bundle.js:521
fn @ bundle.js:76
(anonymous function) @ bundle.js:555__webpack_require__ @ bundle.js:521
(anonymous function) @ bundle.js:544(anonymous function) @ bundle.js:547
```

![Web Inspector Screenshot](https://www.evernote.com/shard/s3/sh/eed9d3f2-95f5-4441-89d0-ac0493ecb532/4bdd9772c118e05c/res/2cee5165-34ac-4deb-af9a-76e4c8ce0f10/skitch.png?resizeSmall&width=832)

![Redux Store Source Code Screenshot](https://www.evernote.com/shard/s3/sh/76e22e36-b8ac-4da1-91af-ef738f9a9391/6fc84558ba832ecb/res/89ebaf1b-0714-49e4-9ead-b3604a6584ac/skitch.png?resizeSmall&width=832)

It seem to clash with `DevTools.instrument()` middleware, maybe a warning should be raised?

**Comment by zalmoxisus** [comment]
Thanks for the report. That's a good suggestion to add a warning, but we have to do that in the Redux DevTools not in the extension. You can run into that also by adding `DevTools.instrument()` store enhancer twice, which actually happens here.

Just FYI, if you want to use a fallback to the vanilla Redux DevTools, use it as follows:
```js
window.devToolsExtension ? window.devToolsExtension() : DevTools.instrument()
```
But you cannot have both at once as it invokes having two devTools intruments.

**Comment by grimen** [comment]
@zalmoxisus Yes, I changed it to that already - but took me a while to notice my mistake, so maybe it would make sense to handle that error with a warning in the console.

**Comment by zalmoxisus** [comment]
@grimen, I know, we have similar issues before. Hopefully, few seconds ago @gaeron released Redux DevTools 3.1.0 with [a descriptive warning in the console](https://github.com/gaearon/redux-devtools/blob/master/src/instrument.js#L363-L364). Thanks for the suggestion!

**Comment by zalmoxisus** [comment]
Implemented in `v1.0.0-beta3`.

**Comment by grimen** [comment]
+1
